integration-cli/docker_cli_import_test.go

```diff
@@ -20,30 +21,20 @@ func (s *DockerSuite) TestImportDisplay(c *check.C) {
 		exec.Command(dockerBinary, "export", cleanedContainerID),
 		exec.Command(dockerBinary, "import", "-"),
 	)
-	if err != nil {
-		c.Errorf("import failed with errors: %v, output: %q", err, out)
-	}
+	c.Assert(err, checker.IsNil)
 
-	if n := strings.Count(out, "\n"); n != 1 {
-		c.Fatalf("display is messed up: %d '\\n' instead of 1:\n%s", n, out)
-	}
-	image := strings.TrimSpace(out)
+	c.Assert(strings.Count(out, "\n"), checker.Equals, 1, check.Commentf("display is expected 1 '\\n' but didn't"))
```

Comment:
There is now a `Count` checker :wink: 
`c.Assert(out, checker.Count, "\n", 1, check.Commentf(…))`